### ISSUE TITLE
extend vpool does not check requirements

### ISSUE THREAD

**Issue by kinvaris** [openvstorage/framework#793]
When trying to extend a vpool, we tried to extend it to a node without WRITE role.
When starting the wizard it was possible to extend the vpool although it should not be.

**Comment by khenderick** [comment]
Observation: It looked like the error message about a missing write role was briefly displayed, but removed as soon as the ip address was loaded.

**Comment by khenderick** [comment]
Fixed by #1038, packaged in openvstorage-2.7.4-rev.4147.6f02d53

**Comment by JeffreyDevloo** [comment]
### Information
Fixed in the GUI but still some issues with API. (https://github.com/openvstorage/framework/issues/1289)

### Test result
Test passed.

### Packages
```
ii  openvstorage                                        2.7.9-rev.4463.0761b63-1       amd64                          openvStorage
ii  openvstorage-backend                                1.7.9-rev.889.63f8d79-1        amd64                          openvStorage Backend plugin
ii  openvstorage-backend-core                           1.7.9-rev.889.63f8d79-1        amd64                          openvStorage Backend plugin core
ii  openvstorage-backend-webapps                        1.7.9-rev.889.63f8d79-1        amd64                          openvStorage Backend plugin Web Applications
ii  openvstorage-core                                   2.7.9-rev.4463.0761b63-1       amd64                          openvStorage core
ii  openvstorage-hc                                     1.7.9-rev.889.63f8d79-1        amd64                          openvStorage Backend plugin HyperConverged
ii  openvstorage-sdm                                    1.6.9-rev.508.9903a29-1        amd64                          Open vStorage Backend ASD Manager
ii  openvstorage-webapps                                2.7.9-rev.4463.0761b63-1       amd64                          openvStorage Web Applications

```
